Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/jobs/JobUtils.java

```diff
@@ -153,11 +150,11 @@ public void openDefaultBrowser(@NotNull final URI uri) throws IOException {
         }
     }
 
-    public void openFileExplorer(@NotNull final File file) throws IOException {
+    public static void openFileExplorer(@NotNull final File file) throws IOException {
```

Comment:
JobUtils is an object passing to WebView of JavaFx before. But it's kind of `helper` class now.